### DISCUSSION TITLE
Responsive breadcrumbs

### DISCUSSION THREAD

**Discussion by AnaMontes11** [Telefonica/mistica-design#657] [Proposals]
En mobile es complicado mostrar la ruta completa de breadcrumbs sin saturar la pantalla pudiendo pasar en algunos casos a dos líneas.

Para solucionarlo, se podría evolucionar el componente para que cuando la pantalla se vuelve más pequeña se muestre solo una opción usando un back button (volviendo a la pág padre inmediata).

De esta manera, los usuarios pueden seguir navegando hacia arriba en la jerarquía.

Ejemplo:
![Captura de Pantalla 2022-08-04 a las 9 21 28](https://user-images.githubusercontent.com/88187691/182790805-a8256f97-91a4-4715-a3a8-41b6f9c9b17d.png)

**Comment by yceballost** [comment]
De momento no lo veo como una necesidad
